CONTRIBUTING.rst

```diff
@@ -1,107 +1,104 @@
-Contributing to Crate
-=====================
+============
+Contributing
+============
 
-Thanks for considering contributing to crate.
+Thank you for considering to make a contribution.
 
-The steps outlined in this document should be seen as guidelines. These are NOT
-strict rules and we won't blame anyone who tries to contribute.  By following
-these things you'll just make our lives a tad bit easier, but if you don't
-follow them we'll still be happy about your contribution.
+This document is a guideline. Don't worry too much about getting everything
+perfect. We are more than happy to work with you on your contribution.
 
-The easiest way to contribute is either by filling issues on `Github
-<https://github.com/crate/crate/issues>`_ or by contributing to our
-documentation.
+Reporting issues it the easiest way to contribute. But we also accept pull
+requests for the code and for the documentation. For more information on how to
+work on the code or documentation, see our `developer guide`_.
 
-For more information on how to do that please refer to the documentation
-section in the ``DEVELOP.rst``.
+If you have any questions, please feel free to ask us on Slack_.
 
-If you have any questions don't hesitate to ask us on Slack_.
+Reporting Issues
+================
 
-.. _Slack: https://crate.io/docs/support/slackin/
+Before you report an issue, please search the existing issues to make sure
+someone hasn't already reported it.
 
-Reporting an issue
-------------------
+When reporting a new issue, include as much detail as possible. For example:
 
-- Search existing issues before raising a new one.
+- Say what you did, what happened, and what you expected to happen
 
-- include as much details as possible. This might include:
+- Provide steps to reproduce the issue
 
-  - Which OS you're using.
+- Say which OS you're using
 
-  - Which version you've been running.
+- Say wjich version of CrateDB you are running
 
-  - Logs/Stacktrace of the error that occurred.
-    (For example ``crash`` the command-line client can be started with the `-v`
-    option to get a stacktrace from the server if a SQL statement resulted in
-    an error)
+- Include logs or stacktraces
 
-  - Steps to reproduce.
+  - For example, the ``crash`` CLI client can be started with the ``-v`` option
+    to get a stacktrace from the server if a SQL statement resulted in an
+    unexpected error.
 
-Pull requests
--------------
+Pull Requests
+=============
 
-Before we can accept any pull requests to Crate we need you to agree to our
-CLA_. Once that is done, we suggest to continue as follows:
+Before we can accept any pull requests we need you to agree to our CLA_.
```

Comment:
I think CLAs are commonly understood. if not, clicking the link gives the justification